modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/components/FieldFeedback.tsx

```diff
@@ -15,30 +15,25 @@ export function FieldFeedback({
 	warningMessage,
 	...otherProps
 }: IProps) {
-	if (!errorMessage && !helpMessage && !warningMessage) {
-		return null;
-	}
+	const message =
+		warningMessage && !errorMessage ? warningMessage : errorMessage;
+
+	const symbol =
+		warningMessage && !errorMessage ? 'warning-full' : 'exclamation-full';
 
 	return (
 		<ClayForm.FeedbackGroup
 			className="lfr-de__field-feedback"
 			{...otherProps}
 		>
-			{errorMessage && (
-				<ClayForm.FeedbackItem id={`${name}_fieldError`} role="alert">
-					<ClayForm.FeedbackIndicator symbol="exclamation-full" />
-
-					{errorMessage}
-				</ClayForm.FeedbackItem>
-			)}
-
-			{warningMessage && !errorMessage && (
-				<ClayForm.FeedbackItem id={`${name}_fieldError`} role="alert">
-					<ClayForm.FeedbackIndicator symbol="warning-full" />
-
-					{warningMessage}
-				</ClayForm.FeedbackItem>
-			)}
+			<ClayForm.FeedbackItem
+				aria-live="assertive"
+				id={`${name}_fieldError`}
+			>
+				{message && <ClayForm.FeedbackIndicator symbol={symbol} />}
+
+				{message}
+			</ClayForm.FeedbackItem>
```

Comment:
There is a pretty large change to the logic here. This change causes the `<ClayForm.FeedbackGroup>` and the `<ClayForm.FeedbackItem>` to always be rendered when previously they were only rendered when if there was a message. What is the reason for this change? Is there any unintended spacing changes that occurs as a result?

Reply:
Thanks @ethib137!
The reason I made this change, was that Screen Reader didn't recognize the `id` on the feedback because it was not rendered yet, and because of that, it did not read the error. 
I haven't noticed any change in spacing but I'll check!

Reply:
There is no space until the message is displayed.

![Screenshot_20240415_085208](https://github.com/liferay-forms/liferay-portal/assets/47724428/cf330aab-e11c-4406-b453-f21565f2ac52)